examples/controls_example/public/react_controls/data_controls/range_slider/components/range_slider.scss

```diff
@@ -22,6 +22,7 @@
 }
 
 .rangeSliderAnchor__fieldNumber {
+  min-width: auto !important;
```

Comment:
This fixes a bug where the step arrows were being cut off:

| Before | After |
|--------|--------|
| ![image](https://github.com/elastic/kibana/assets/8698078/69d20be6-0b53-4f94-906e-e527c1e5fbe2) | ![image](https://github.com/elastic/kibana/assets/8698078/ba48326f-c4e7-4444-8e55-7e50150aa703) |